locations/storefinders/lighthouse.py

```diff
@@ -1,49 +1,122 @@
 
-# from locations.categories import Categories
-# from locations.hours import DAYS_EN, OpeningHours
-
+# Lighthouse e-commerce platform store finder, with official URL of
 # https://www.lighthouse.gr/
 #
-# Detected by:
-# meta name="generator" content="Created by Lighthouse, powered by VENDD e-commerce platform - www.lighthouse.gr"
-# data-control="box"
+# This store finder is detected via a HTML node:
+#   <meta name="generator" content="Created by Lighthouse, powered by VENDD
+#   e-commerce platform - www.lighthouse.gr" />
+# As well as HTML nodes for each location/feature:
+#   <article data-control="box" data-latitude=".." data-longitude="..">
+#
+# To use this spider, specify a 'start_url' for the store finder page. If
+# opening hours are provided, also specify 'days' as a language-specific list
+# of days as specified in locations/hours.py (such as DAYS_GR). If 'days' is
+# not specified, an attempt will be made to automatically detect the language.
+#
+# The following methods can be overridden:
+# 1. parse_extras
+#      Override this method to provide customised feature tag extraction of
+#      facts such as whether the feature has a publicly accessible toilet. By
+#      default, known feature tags are automatically extracted.
+# 2. parse_opening_hours
+#      Override this method to provide customised opening hours extraction
+#      and parsing, should this be necessary. By default, opening hours are
+#      extracted by guessing the language these opening hours are provided in.
+#      Use the 'days' attribute to specify the language explicitly.
+# 3. parse_item
+#      Override this method to clean up extracted data such as location names
+#      with unwanted suffixes.
 
 
 class LighthouseSpider(Spider):
-    def parse(self, response, **kwargs):
-        for location in response.xpath("//article[@data-control='box']"):
+    days: dict = None
+
+    def parse(self, response: Response):
+        for location in response.xpath('//article[@data-control="box"]'):
             item = Feature()
             item["ref"] = location.xpath("@id").get()
             item["lat"] = location.xpath("@data-latitude").get()
             item["lon"] = location.xpath("@data-longitude").get()
-            item["name"] = location.xpath("*[contains(@class, 'name-container')]/span/text()").get()
-            item["street_address"] = location.xpath(
-                "*[contains(@class, 'info-container')]/*[contains(@class, 'address-one')]/text()"
-            ).get()
-            phones = location.xpath("*[contains(@class, 'info-container')]/*[contains(@class, 'phone')]/text()").get()
-            if phones is not None:
-                item["phone"] = phones.split(",")[0]
+            item["name"] = (
+                location.xpath('.//*[contains(@class, "name") and not(contains(@class, "name-"))]/text()')
+                .get("")
+                .strip()
+            )
+            item["addr_full"] = ", ".join(
```

Comment:
Correct usage is `item["addr_full"] = clean_address(["Line 1", "Line 2", "Line 3"])`